api-gateway-config/scripts/lua/lib/redis.lua

```diff
@@ -411,39 +382,22 @@ function _M.subscribe(redisSubClient, redisGetClient)
       local redisKey = utils.concatStrings({resourcePrefix, ":", tenant, ":", gatewayPath})
       -- Don't allow single quotes in the gateway path
       if string.match(gatewayPath, "'") then
-        logger.debug(utils.concatStrings({"Redis key \"", redisKey, "\" contains illegal character \"'\"."}))
+        logger.info(utils.concatStrings({"Redis key \"", redisKey, "\" contains illegal character \"'\"."}))
```

Comment:
It notes that the redis key that was created has an illegal character and ignores it, but continues with the subscribe